test/core/byte.test.js

```diff
@@ -1,21 +1,46 @@
 "use strict";
+/*global context, before, after*/
```

Comment:
Rather than using `/* global` please just add these globals to https://github.com/jneen/parsimmon/blob/master/test/.eslintrc#L5

Reply:
For this I actually just added the mocha env to eslint.  I just forgot to remove this line.

Reply:
oh that's much better, thanks